src/commcare_cloud/ansible/roles/elasticsearch/tasks/main.yml

```diff
@@ -150,7 +161,6 @@
     - "{{ elasticsearch_data_dir }}"
     - "{{ elasticsearch_data_dir }}/data"
     - "{{ elasticsearch_data_dir }}/logs"
-    - "{{ elasticsearch_data_dir }}/tmp"
```

Comment:
Curious why the `es_conf` tag was not added to this task? Does the conf file not reference data and logs directories?

Reply:
I want just thinking about java options file and we recently updated 🤔 
I guess having it here would also make sense since it can also be updated by updating es_conf.